src/server/req_runjob.c

```diff
@@ -985,11 +985,15 @@ svr_strtjob2(job *pjob, struct batch_request *preq)
 
 	/* send the job to MOM */
 
+	/* clearing old comment since we are sending a fresh job */
+	job_attr_def[(int) JOB_ATR_Comment].at_decode(
+			&pjob->ji_wattr[(int) JOB_ATR_Comment], (char *) 0,
+			(char *) 0, "");
```

Comment:
With the server in charge of setting the comment after run, we have the opportunity to be more clear than we had before.  Instead of just saying, "Job is about to run" we can say things like, "Job is staging files" or "Nodes are being provisioned for job".  I dislike just unsetting the comment.  An unset comment has meaning today.  It means the scheduler has not seen the job.